inference-engine/src/low_precision_transformations/src/markup_precisions.cpp

```diff
@@ -0,0 +1,217 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "low_precision/markup_precisions.hpp"
+
+#include <memory>
+#include <unordered_set>
+#include <set>
+#include <vector>
+
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include "low_precision/network_helper.hpp"
+#include "low_precision/rt_info/precisions_attribute.hpp"
+#include "low_precision/rt_info/precision_preserved_attribute.hpp"
+
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::low_precision::MarkupPrecisions, "MarkupPrecisions", 0);
+
+ngraph::pass::low_precision::MarkupPrecisions::MarkupPrecisions(const std::vector<OperationPrecisionRestriction>& restrictions) {
+    for (const OperationPrecisionRestriction& restriction : restrictions) {
```

Comment:
Please use `auto` instead of this long names. It really complicates reading.

Reply:
done, thanks